resolwe/flow/views/mixins.py

```diff
@@ -102,7 +103,55 @@ def update(self, request, *args, **kwargs):
                         'Invalid descriptor_schema slug "{}" - object does not exist.'.format(ds_query)]},
                     status=status.HTTP_400_BAD_REQUEST)
 
-        return super(ResolweUpdateModelMixin, self).update(request, *args, **kwargs)
+        # return super(ResolweUpdateModelMixin, self).update(request, *args, **kwargs)
```

Comment:
Add a comment above this line, saying that "Use the original method instead when support for locking is added" + link to issue (as below). Otherwise it is not immediately clear why there is a commented-out block of code here.